scripts/build.sh

```diff
@@ -175,7 +175,7 @@ case "$platform" in
 
   catalyst)
     xcb_flags=("${catalyst_flags[@]}")
-    destination='platform="macOS,variant=Mac Catalyst,arch=x86_64" TARGETED_DEVICE_FAMILY=2'
+    destination='platform="macOS,variant=Mac Catalyst,name=Any Mac'
```

Comment:
I couldn't get this to work (locally at least) without adding `name=Any Mac`; otherwise says `The requested device could not be found because no platform could be found for the requested platform name`. Found this approach in https://stackoverflow.com/a/63526150.